client/src/docs-ui/page/redirects.ts

```diff
@@ -55,7 +55,9 @@ export const redirects = {
   "#usage-4": "/searchable#usage",
   "#known-limitations": "/searchable#known-limitations",
   "#backfill-your-elasticsearch-index-from-your-dynamodb-table":
-    "/searchable#backfill-your-elasticsearch-index-from-your-dynamodb-table",
+    "/searchable#backfill-your-opensearch-index-from-your-dynamodb-table",
+  "/searchable#backfill-your-elasticsearch-index-from-your-dynamodb-table":
+    "/searchable#backfill-your-opensearch-index-from-your-dynamodb-table",
```

Comment:
The majority of this redirects objects doesn't seem to be used anywhere in the codebase.

We'll add these particular redirects to the rewrite rules list in amplify console for server-side redirects.